dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogInterceptor.kt

```diff
@@ -102,17 +104,24 @@ internal constructor(
         { AndroidTracer.Builder().build() }
     )
 
+    init {
+        if (!RumFeature.isInitialized()) {
```

Comment:
Maybe we should add one also for the `TracingInterceptor` -> `if(!TracingFeature.isInitialized())...`

Reply:
It's already there in the Tracing Interceptor